.github/workflows/ci.yaml

```diff
@@ -22,10 +20,15 @@ jobs:
         - ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with: {}
```

Comment:
Is this bit required? Or was it just for completion? I don't remember if `cue` complained.

Reply:
It's not. Once the final fixes came through, it was removed by the exporting.